src/Makefile.am

```diff
@@ -471,6 +471,10 @@ if HARDEN
 	$(AM_V_at) READELF=$(READELF) OBJDUMP=$(OBJDUMP) $(top_srcdir)/contrib/devtools/security-check.py < $(bin_PROGRAMS)
 endif
 
+check-formatting:
+	@echo "Checking source formatting style..."
+	(cd $(top_srcdir)/src; ../contrib/devtools/clang-format.py check clang-format-3.8 `cat ./.formatted-files`)
```

Comment:
Great, probably Travis crew add the `trusty-update` repo to their servers 
http://packages.ubuntu.com/search?keywords=clang-format&searchon=names&suite=trusty-updates&section=all